packages/manager/apps/telecom/src/app/telecom/pack/index.js

```diff
@@ -1,16 +1,43 @@
 import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
 
-import component from './packs.component';
-import routing from './packs.routing';
+const moduleName = 'ovhManagerPacksLoading';
 
-import dashboard from './dashboard/pack.module';
+angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider
+      .state('telecom.packs', {
+        url: '/pack',
+        redirectTo: 'telecom.packs.index',
+        views: {
+          'telecomView@telecom': {
+            template: '<div ui-view=></div>',
```

Comment:
not sure if it's a valid syntax to have an empty attribute value without quotes, maybe just remove the equal sign?

Reply:
I think it's a typo,  nice catch